src/Builder/ListBuilder.php

```diff
@@ -121,9 +121,9 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
 
             if (null === $template) {
                 if (ClassMetadataInfo::ONE == $fieldDescription->getMappingType()) {
-                    $template = 'SonataDoctrineMongoDBAdminBundle:CRUD:list_mongo_one.html.twig';
+                    $template = 'SonataAdminBundle:CRUD/Association:list_many_to_one.html.twig';
```

Comment:
Shouldn't you need the `@` prefix?

Reply:
@core23 no as this is still the way with `:` instead of `@`.

Reply:
should that be changed also?

Reply:
Would be nice, if we use one way for alle template paths (new or old one).

Also we need a a twig dependency in `composer.json`.

Reply:
@core23 Well if the point is to use only one way then we should stick with `:` as that is used all over bundle or switch to `@`?

Reply:
also switching to `@` should probably a different PR?

Reply:
Doing it here would be nice if you can :)

Reply:
Replace all templates paths in bundle with `@` ? @jordisala1991

Reply:
Not all, but the ones that you are changing yes